### PR TITLE
devsign-server

### PR DESCRIPTION
**Pull request by crperezchica**

### CODE REVIEW

File: .env [hunk 1]
```diff
@@ -0,0 +1,7 @@
+MONGODB_URI=mongodb://127.0.0.1:27017/devsign
```
Comment (mehtaphysical) [inline .env]: you should gitignore this file and remove it from github

File: test/tweets.test.js [hunk 2]
```diff
@@ -0,0 +1,29 @@
+require('dotenv').config();
+require('../lib/utils/connect')();
+
+const mongoose = require('mongoose');
+const seedData = require('./seedData');
+
+const request = require('supertest');
+const app = require('../lib/app');
+
+jest.mock('../lib/services/auth.js');
```
Comment (mehtaphysical) [inline test/tweets.test.js]: also create a ensureAuth mock and use it here

File: test/tweets.test.js [hunk 3]
```diff
@@ -0,0 +1,29 @@
...
+
+describe('tweets routes', () => {
+  beforeEach(() => {
+    return seedData(100);
+  });
+
+  afterEach(() => {
+    return mongoose.connection.dropDatabase();
+  });
```
Comment (mehtaphysical) [inline test/tweets.test.js]: afterAll `mongoose.connection.close()`

File: lib/routes/tweets.js [hunk 4]
```diff
@@ -0,0 +1,45 @@
...
+  })
+
+
+  .get('/', ensureAuth(), (req, res, next) => {
+    Tweet
+      .find()
+      .lean()
+      .then(tweets => populateUsers(tweets))
+      .then(tweets => res.send(tweets))
+
```
Comment (MariahAdams) [inline lib/routes/tweets.js]: Keep whitespace consistent across files.

**Pull request merged**